### PR TITLE
fix(devops): fixed staging tag issue on github action

### PR DESCRIPTION
**Pull request by cnlangzi**
Tasks to complete before merging PR:
- [x]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zwalletcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [x]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a branch at [0chain/system_test](https://github.com/0chain/system_test)

### CODE REVIEW

File: .github/workflows/build-zwallet.yaml [hunk 1]
```diff
@@ -66,6 +66,8 @@ jobs:
           echo "TAG=v0.0.0-master" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
         } ElseIf ( "${{github.base_ref}}" -eq "dev" -OR "${{github.ref}}" -eq "refs/heads/dev" ){
           echo "TAG=v0.0.0-dev" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
+        } ElseIf ( "${{github.base_ref}}" -eq "staging" -OR "${{github.ref}}" -eq "refs/heads/staging" ){
+          echo "TAG=v0.0.0-staging" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
```
Comment (stewartie4) [inline .github/workflows/build-zwallet.yaml:70]: does this say that the staging latest tag will no longer be called "staging"? what's the reason for the change?

Comment (cnlangzi) [reply .github/workflows/build-zwallet.yaml:70]: TAG is for release page . if it is staging , you will see the duplicated staging issue on https://github.com/0chain/zwalletcli/releases/tag/staging

do you still remember we have duplicated staging on git repo
one is branch, another is tag. 

this PR fixes it.

Comment (stewartie4) [reply .github/workflows/build-zwallet.yaml:70]: ah yes i remember that issue now, good fix

**Review by stewartie4 — APPROVED** [review]
